tools/sof-tplgreader.py

```diff
@@ -79,12 +79,17 @@ def loadFile(self, filename, sofcard=0):
                     self._pipeline_lst.append(pb_pipeline_dict)
                 self._pipeline_lst.append(pipeline_dict)
 
-        # format pipeline, this change for script direct access 'rate' 'channel' 'dev'
+        # format pipeline, this change for script direct access 'rate' 'channel' 'dev' 'snd'
         for pipeline in self._pipeline_lst:
             #pipeline['fmt']=pipeline['fmt'].upper().replace('LE', '_LE')
             pipeline['rate'] = pipeline['rate_min'] if int(pipeline['rate_min']) != 0 else pipeline['rate_max']
             pipeline['channel'] = pipeline['ch_min']
             pipeline['dev'] = "hw:" + str(sofcard) + ',' + pipeline['id']
+            pipeline['snd'] = "/dev/snd/pcmC" + str(sofcard) + "D" + pipeline['id']
```

Comment:
Please add a comment upon this line to tell the reader that the string takes this form: /dev/snd/pcmC + card_number + D + device number + capability. so they know why you construct the string like this. the comment can be like this:`the character devices for PCM under /dev/snd take the form of "pcmC + card_number + D + device_number + capability", eg, pcmC0D0p.`

Reply:
good idea